packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -138,6 +163,26 @@ export class SelectPageObject {
         );
     }
 
+    public pressCharacterKey(character: string): void {
+        if (character.length !== 1) {
+            throw new Error(
+                'character parameter must contain only a single character'
+            );
+        }
+
+        if (
+            this.selectElement.open
+            && this.selectElement.filterMode !== FilterMode.none
+        ) {
+            const filterInput = this.getFilterInput();
+            filterInput!.value = filterInput!.value + character;
```

Comment:
nit: Why not put the `!` on the line above so it's only needed once?

```suggestion
            const filterInput = this.getFilterInput()!;
            filterInput.value = filterInput.value + character;
```